src/ImageCrop/Interfaces/ImageCropDataAccessInterface.php

```diff
@@ -71,4 +71,11 @@ public function getImageCropById($cropId, $languageId);
      * @throws ImageCropDataAccessException
      */
     public function getExistingCrops(CmsMediaDataModel $cmsMedia);
+
+    /**
+     * @throws ImageCropDataAccessException
+     *
+     * @param string $cropId
+     */
+    public function deleteCrop($cropId);
```

Comment:
Please add argument type declaration and return value.

Reply:
Forgot that we require PHP 7.1 now.